dashi/poolqc/pooling_qc_sample.py

```diff
@@ -26,62 +29,136 @@
+                              labelStyle={'paddingLeft': 30}
+
+                          ),
+                      ),
+                      html.Div([
+                          html.P(children='Library Type',
+                                 style={'padding-left': '5px', 'font-weight': 'bold'}),
+                          dcc.Dropdown(
+                              id='sample_type',
+                              options=[{'label': 'DNA: WG', 'value': 'WG'},
+                                       {'label': 'DNA: EX', 'value': 'EX'},
+                                       {'label': 'DNA: TS', 'value': 'TS'},
+                                       {'label': 'DNA: AS', 'value': 'AS'},
+                                       {'label': 'RNA: MR', 'value': 'MR'},
+                                       {'label': 'RNA: SM', 'value': 'SM'},
+                                       {'label': 'RNA: WT', 'value': 'WT'},
+                                       {'label': 'RNA: TR', 'value': 'TR'}],
+                              value=['WG', 'EX', 'TS', 'AS', 'MR', 'SM', 'WT', 'TR'],
+                              clearable=False,
+                              multi=True
+                          )])
+                  ])]
+              ),
+    sd.RaisedButton(id='filters',
+                    label='Report Filters'),
+    html.Div(children=[
+        dcc.ConfirmDialog(
+            id='warning',
+            message='The selected run does not return any data. Analysis may have not been completed yet.' '''
         '''' Click either "Ok" or "Cancel" to return to the most recent run.'
-    ),
-    dcc.Location(
-        id='run_url',
-        refresh=False
-    ),
-    html.Div(children=
-    dcc.Dropdown(
-        id='select_a_run',
-        options=[{'label': r, 'value': r} for r in runs],
-        value=runs[0],
-        clearable=False
-    )),
-    html.Div(
-        dcc.Dropdown(
-            id='lane_select'
-        )),
-    html.Div(id='Title', style={'fontSize': 25, 'textAlign': 'center', 'padding': 30}),
-    html.Div(children=''),
-    html.Div(
-        dcc.Graph(id='SampleIndices'),
-    ),
-    html.A(
-        'Download Data',
-        id='download-link',
-        download='rawdata.csv',
-        href='',
-        target='_blank'
-    ),
-    html.Div(
-        dt.DataTable(
-            id='Summary Table',
-            style_cell={
-                'minWidth': '150px',
-                'textAlign': 'center'
-            },
-            style_table={
-                'maxHeight': '1000px',
-                'maxWidth': '100%',
-                'overflowY': 'scroll',
-                'overflowX': 'scroll'
-            },
-            style_header={'backgroundColor': 'rgb(222,222,222)',
-                          'fontSize': 16,
-                          'fontWeight': 'bold'},
-
-        )),
-]
-)
+        ),
+        dcc.Location(
+            id='PBQCrun_url',
+            refresh=False
```

Comment:
Comment out the `Location` functionality (and it's callback), as it suffers the same bug as `bcl2fastq`.